src/pipeline/processors/technicals.py

```diff
@@ -32,8 +32,9 @@ def process(self, context: SharedContext, candle: Candle):
         cache_reader = CandleCache.context_reader(context)
         symbol_candles = cache_reader.get_symbol_candles(candle.symbol) or []
 
-        if candle.timestamp in list(map(lambda candle: candle.timestamp, symbol_candles)):
-            candles = symbol_candles
+        candle_index = [x for x in range(len(symbol_candles)) if symbol_candles[x].timestamp == candle.timestamp] or None
+        if candle_index and candle_index[0] > 0:
```

Comment:
There might be an issue when candle_index[0] is 0, review the logic, perhaps add supporting tests if this is the way we go.